sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> True, that could be ambiguous. Then maybe we could just introduce the option NETCONF_OPT_CCA for CCA. CSMA could be de/activated by setting the options minBE, maxBE and CSMA_RETRIES. Do you see any option clash there?

No option clash, but CSMA should be explicitly be enabled or disabled, not via CSMA_RETRIES. And what would `NETCONF_OPT_CCA` actually do? In this case `AUTOCCA` is more understandable.

> In my currently open CSMA_PR (#2467) this is handled by introducing the new event-types NETDEV_EVENT_CCA_CHANNEL_BUSY/IDLE. The driver has to figure out according to the occured interrupts what event it needs to trigger.

There is already `NETCONF_OPT_IS_CHANNEL_CLR` to manually sense the channel. Otherwise I find it more logical to give feedback like `NETDEV_EVENT_TX_MEDIUM_BUSY` when sending was not possible (i.e. channel not clear) or just `NETDEV_EVENT_TX_COMPLETE` if the frame was sent out. (that's how I've done [it](https://github.com/daniel-k/RIOT/blob/4cf2f53010a975e650f4093b757ef2c043c053d2/sys/net/link_layer/ng_lwmac/ng_lwmac.c#L392))